src/runtime/Module.h

```diff
@@ -34,6 +34,13 @@ class JITModule;
 
 struct WASMParsingResult;
 
+enum JITFlagValue : uint32_t {
+    useJIT = 1 << 0,
```

Comment:
What is the purpose of this flag?

Reply:
https://github.com/Samsung/walrus/blob/5c0b91eb3d70eee23d3b54dad4da01e87234c1f8/src/parser/WASMParser.cpp#L2576